components/lib/crash-sentry/src/main/java/mozilla/components/lib/crash/sentry/SentryService.kt

```diff
@@ -76,6 +79,9 @@ class SentryService(
     }
 
     override fun report(throwable: Throwable, breadcrumbs: ArrayList<MozillaBreadcrumb>): String? {
+        if (!sendCaughtExceptions) {
+            return null
+        }
```

Comment:
Just to clarify, we are adding `sendCaughtExceptions` to avoid misuses and sending caught unless the consumer is completely aware that want to send the report?

Reply:
Yup.